templates/base.html

```diff
@@ -0,0 +1,53 @@
+<html>
+
+<head>
+	<meta http-equivalent="Content-Type" content="text/html; charset=utf-8" />
+	<link rel="stylesheet" type="text/css" href={{ url_for( 'static', filename='style.css' ) }}/>
```

Comment:
This should be `style.min.css` now, as it should be in all the template files (and similarly in the static ones), otherwise we get no benefits from this.

Reply:
My bad, it was changed locally but I forgot to check it in.